index.html

```diff
@@ -10,229 +10,233 @@
-                    <li>RSpec</li>
-                    <li>Capybara</li>
-                    <li>Selenium</li>
-                </ul>
-            </li>
-            <li class="card">
-                <img class="screenshot" src="/Images/skills-icon.svg" alt="Skills">
-                <h3>Skills</h3>
-                <ul class="tags">
-                    <li>Codekit</li>
-                    <li>GitHub</li>
-                    <li>codepen</li>
-                    <li>Gitlab</li>
-                    <li>Termial</li>
-                </ul>
-            </li>
-        </ul>
-    </section>
-    <section class="form">
-        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-
-        <form 
-            action="https://formspree.io/f/xlezgrbo"
-            method="POST">
-            <input type="text" name="fullname" placeholder="Full name" maxlength='30' autocomplete="name" required>
-            <input type="email" name="email" placeholder="Email Addres" autocomplete="email" required>
-            <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
-            <button type="submit" class="button">Get in touch</button>
-        </form>
-    </section>
-    <footer>
-        <ul class="network">
-            <li class="social">
-                <a target="_blank" href="https://github.com/Kakalanp" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="g"/></a>
-            </li>
-            <li class="social">
-                <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="li"/></a>
-            </li>
-            <li class="social">
-                <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="t"/></a>
-            </li>
-        </ul>
-        <a href="mailto:jgarridoleccazanetti@gmail.com" class="email">jgarridoleccazanetti@gmail.com</a>
-    </footer>
+        <section id="form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+            <form 
+                action="https://formspree.io/f/xlezgrbo"
+                method="POST">
+                <input type="text" name="fullname" placeholder="Full name" maxlength='30' autocomplete="name" required>
+                <input type="email" name="email" placeholder="Email Addres" autocomplete="email" required>
+                <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                <button type="submit" class="button">Get in touch</button>
+            </form>
+        </section>
```

Comment:
- [ ] You also added much space between the about projects and the form. I will suggest you reduce the space.
- [ ] Also place the form button to the left. It's currently at the center

Your Design | 
:---------------:|
![e10](https://user-images.githubusercontent.com/39523312/149405480-0a638517-8fc1-4601-8364-e420b81b38c0.png) 

 Figma Design |
:-------------------:|
![e9](https://user-images.githubusercontent.com/39523312/149405492-40090d12-1fc5-474c-9397-62a24f1b3e51.png)

Reply:
Once again, I think these changes look better the way I did them, that's why they're there.

Reply:
Also, the view you're showing is for 922px